wrappers/juce/SC3Editor.cpp

```diff
@@ -78,6 +78,10 @@ SC3AudioProcessorEditor::~SC3AudioProcessorEditor() {
     actiondata ad;
     ad.actiontype = vga_closeeditor;
     sendActionToEngine(ad);
+
+    if (audioProcessor.mNotify == this)
```

Comment:
A few lines up it sets this to 0 anyway, so doubt this will do anything

Reply:
Oh you are right. We just need the line below